2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,17 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arr) {
+  return arr.filter((element, index) => index < 5);
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  return arr.sort();
 }
```

Comment:
Some tests for this function are failing because it modifies the passed in array. I'd suggest making a copy of the array and sorting that, e.g `[...arr].sort();`